apps/api/src/app/subscribers/usecases/remove-subscriber/remove-subscriber.spec.ts

```diff
@@ -51,10 +47,9 @@ describe('Remove Subscriber', function () {
           organizationId: session.organization._id,
         })
       );
-      throw new Error('Should not reach here');
+      expect(true, 'Should never reach this statement').to.be.false;
```

Comment:
A small refactoring without throwing an error that will trigger the catch block again.